pandas/tests/indexing/test_iloc.py

```diff
@@ -32,7 +32,7 @@
 )
 import pandas._testing as tm
 from pandas.api.types import is_scalar
-from pandas.tests.indexing.common import Base
+from pandas.tests.indexing.common import check_result
```

Comment:
maybe rename to assert_result (or something slightly more verbose)

Reply:
Renamed to `check_indexing_smoketest_or_raises`